test/testlib/nuodb_admin_utilities.go

```diff
@@ -303,3 +303,22 @@ func AwaitServerState(t *testing.T, namespace string, adminPod string,
 		return false
 	}, timeout)
 }
+
+// VerifyLables checks if adminPod in namespace has label and its value is one of the expectedValues.
+func VerifyLables(t *testing.T, namespace string, adminPod string, label string, expectedValues []string) {
+	fieldName := "labels." + label
+	output, err := k8s.RunKubectlAndGetOutputE(t, k8s.NewKubectlOptions("", "", namespace), "exec", adminPod, "-c", "admin", "--",
+		"nuocmd", "--show-json-fields", fieldName, "get", "servers")
+	require.NoError(t, err, output)
+
+	lines := strings.Split(output, "\n")
+	require.Equal(t, 3, len(lines))
+
+	var actualField string
+	var actualValue string
+	read, err := fmt.Sscan(lines[1], &actualField, &actualValue)
```

Comment:
This would be easier to read if we do JSON deserialization. There is already a function that deserializes the entire server object:
https://github.com/nuodb/nuodb-helm-charts/blob/master/test/testlib/nuodb_admin_utilities.go#L259

If you add a `Labels` field to the [`NuoDBServer` struct](https://github.com/nuodb/nuodb-helm-charts/blob/master/test/testlib/NuoDBServer.go#L29), then it would be easy to check for the presence of several labels.